samples/best_practices/disallow_host_network_hostport.yaml

```diff
@@ -1,24 +1,33 @@
 apiVersion: kyverno.io/v1alpha1
 kind: ClusterPolicy
 metadata:
-  name: validate-host-network-hostport
+  name: host-network-port
   annotations:
     policies.kyverno.io/category: Security
     policies.kyverno.io/description: Using 'hostPort' and 'hostNetwork' allows pods to share 
       the host network stack, allowing potential snooping of network traffic from an application pod.
 spec:
   rules:
-  - name: validate-host-network-hostport
+  - name: validate-host-network
     match:
       resources:
         kinds:
         - Pod
     validate:
-      message: "Defining hostNetwork and hostPort are not allowed"
+      message: "Use of hostNetwork is not allowed"
+      pattern:
+        spec:
+          =(hostNetwork): false
+  - name: validate-host-port
+    match:
+      resources:
+        kinds:
+        - Pod
+    validate:
+      message: "Use of hostPort is not allowed"
       pattern:
         spec:
-          (hostNetwork): false
           containers:
           - name: "*"
-            ports:
-            - hostPort: null
+            =(ports):
```

Comment:
as explained in issue #461,
Having the equality anchor doesn't add any conditions here. ( as it expects the key should have the specified value)

Reply:
@JimBugwadia 
we can also remove the equality anchor here to simplify the policy conditions.

Reply:
That would require ports - which is optional.

Reply:
Got it